modules/ROOT/pages/reconnection.adoc

```diff
@@ -148,7 +148,7 @@ Connection connection = connectionProvider.connect();
   try {
     connection.startStreaming();
   } catch(Exception e){
-    throw new ConnectionException(e, connection); //<1>
```

Comment:
We should also modify the text that is above, which states that the exception must be thrown just like in operations.